src/components/Navigation.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Link, useHistory, useLocation } from 'react-router-dom'
+import { Navbar, NavbarBrand, Nav, NavItem, NavLink } from 'reactstrap'
+
+const Navigation = () => {
+    const history = useHistory()
+    const location = useLocation()
+
+    const logout = e => {
+        e.preventDefault()
+        localStorage.clear()
+        history.push('/login')
+    }
+
+    return (
+        <Navbar>
+            <NavbarBrand tag={Link} to='/' className='mr-auto navbar-brand'>
+                Artificial Artist
+            </NavbarBrand>
+
+            {/* display create videos and log out if user has token, else 
+            display browse videos, log in, and sign up */}
+            {localStorage.getItem('token') ? (
+                <Nav className='nav-links'>
+                    <NavItem>
+                        <NavLink
+                            tag={Link}
+                            to='/create'
+                            className={location.pathname === '/create' ? 'active' : ''}
+                        >
+                            Create Videos
+                        </NavLink>
+                    </NavItem>
```

Comment:
We should consider turning these into their own components, although there might not be so many that it's worth the effort.